libdrgn/language.c

```diff
@@ -67,7 +67,8 @@ struct drgn_error *drgn_language_from_die(Dwarf_Die *die, bool fall_back,
 					  const struct drgn_language **ret)
 {
 	Dwarf_Die cudie;
-	if (dwarf_cu_info(die->cu, NULL, NULL, &cudie, NULL, NULL, NULL, NULL))
+	// if (dwarf_cu_info(die->cu, NULL, NULL, &cudie, NULL, NULL, NULL, NULL))
+	if (!dwarf_cu_die(die->cu, &cudie, NULL, NULL, NULL, NULL, NULL, NULL))
```

Comment:
From what I can tell, `dwarf_cu_info()` and `dwarf_cu_die()` return the same CU DIE, so we can probably make this change unconditionally, right?